### PR TITLE
Create first game

### PR DESCRIPTION
**Pull request by PZalles**

### CODE REVIEW

File: js/scripts.js [hunk 1]
```diff
@@ -1,11 +1,147 @@
...
+    player.score = computer.score = 0;
+    gameState = 'started';
+    setGameElements();
+
+    playerNameElem.innerHTML = player.name;
+  }
+
+}
+
+function playerPick(playerPick) {
```
Comment (matwjablonski) [inline js/scripts.js]: what will do this function? you have duplicated names of functions ;)

File: js/scripts.js [hunk 2]
```diff
@@ -1,11 +1,147 @@
...
+    playerPickElem.innerHTML = playerPick;
+    computerPickElem.innerHTML = computerPick;
+}
+
+function checkRoundWinner(playerPick, computerPick) {
+  playerResultElem.innerHTML = computerResultElem.innerHTML = '';
+
+  var winnerIs = 'player';
+
+    if (playerPick == computerPick) {
```
Comment (matwjablonski) [inline js/scripts.js]: always when you want to compare to variables you should compare also type of this variables

File: js/scripts.js [hunk 3]
```diff
@@ -1,11 +1,147 @@
...
+        playerResultElem.innerHTML = "Win!";
+        player.score++;
+    } else if (winnerIs == 'computer') {
+        computerResultElem.innerHTML = "Win!";
+        computer.score++;
+    }
+
+}
+
+function playerPick(playerPick) {
```
Comment (matwjablonski) [inline js/scripts.js]: third time?

File: js/scripts.js [hunk 4]
```diff
@@ -1,11 +1,147 @@
...
+
+    checkRoundWinner(playerPick, computerPick);
+
+    setGamePoints();
+}
+
+function setGamePoints() {
+    playerPointsElem.innerHTML = player.score;
+    computerPointsElem.innerHTML = computer.score;
+	setTimeout(checkGameEnd, 0);
```
Comment (matwjablonski) [inline js/scripts.js]: why timeout?

**Review by matwjablonski — CHANGES_REQUESTED** [review]

File: js/scripts.js (hunk 1 above)
Comment (PZalles) [reply js/scripts.js]: It display player pick in console. Deleted it.

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**